app/assets/json/translations/react/en.json

```diff
@@ -598,8 +598,16 @@
   "label.dontWantPreference": "I don't want this lottery preference",
   "label.downloadApplication": "Download Application",
   "label.downloadApplicationLang": "Download %{lang} Application",
+  "label.editAlternateContact": "Edit alternate contact information",
```

Comment:
not sure if I need these

Reply:
I think we are safe to remove these. We can pull the translations over when we migrate to React